assets/src/edit-story/output/utils/styles.js

```diff
@@ -27,6 +27,14 @@ function isHexColorString(s) {
 function CustomStyles() {
   const safeToFullRatio = PAGE_RATIO / FULLBLEED_RATIO;
   const fullToSafeRatio = 1 / safeToFullRatio;
+  const safeRatio = PAGE_RATIO;
+  const fullRatio = FULLBLEED_RATIO;
+
+  // The grid layer should stick to the top when the device ratio is between
+  // 493 / 1000 and 9 / 16. The ratio 493 / 1000 can guarantee the grid layer
+  // will respond and not having overlaps with the share button.
+  const gridLayerExpandLowerBound = '493 / 1000';
```

Comment:
Why 493 (e.g. not 492 or 494)? 

Since icon height is static, this ratio will only work for certain screen widths. What is icon height? What is the minimum screen width that this ratio will work for?

Reply:
Got it and updated.